.github/workflows/tests.yml

```
@@ -356,6 +356,19 @@ jobs:
           HOMEBREW_BUILDPULSE_ACCOUNT_ID: 1503512
           HOMEBREW_BUILDPULSE_REPOSITORY_ID: 53238813
 
+      - id: junit_xml
+        run: |
+          cd "${{ steps.set-up-homebrew.outputs.repository-path }}"
+          filenames=$(find test/junit -name 'rspec*.xml' -print | tr '\n' ',')
+          echo "filenames=${filenames%,}" >> "$GITHUB_OUTPUT"
```

(Optional) This might work as a one liner:

```suggestion
          echo "filenames=$(find test/junit -name 'rspec*.xml' -print | paste -s -d ',' -)" >> "$GITHUB_OUTPUT"
```

---

Would prefer as too lines to easier to debug if one fails and the other does not but: thanks for the (GitHub) suggestion!